components/Navigation/MobileNav/MobileNav.module.scss

```diff
@@ -90,11 +90,16 @@
     cursor: pointer;
     background-color: var(--color--white);
     border-radius: 50%;
+    transition: background-color $transition;
 
     @include media-query(menu-break) {
         display: none;
     }
 
+    @include hcm() {
+        filter: invert(1);
```

Comment:
These filters could also possibly have been better handled with system colours, but again I think it is more important to be consistent with the main site.